### PR TITLE
Dev2.36.0

### PR DESCRIPTION
**Pull request by dpaulson45**

### CODE REVIEW

**Comment by dpaulson45** [comment]
@lusassl-msft can you review and test this out.

**Comment by lusassl-msft** [comment]
Sure. Will check it very soon!

File: HealthChecker.ps1 [hunk 1]
```diff
@@ -530,12 +530,38 @@ Function Invoke-CatchActions{
...
 
+Function Test-IsCurrentVersion {
+param(
+[Parameter(Mandatory=$true)][string]$CurrentVersion,
+[Parameter(Mandatory=$true)][string]$TestingVersion
+)
+    Write-VerboseOutput("Calling: Test-IsCurrentVersion")
+    $splitCurrentVersion = $CurrentVersion.Split(".")
+    $splitTestingVersion = $TestingVersion.Split(".")
+    if($splitCurrentVersion.Count -eq $splitTestingVersion.Count)
```
[the remark this answers is not in the document]
Comment (lusassl-msft) [reply HealthChecker.ps1]: I've been thinking about it. This only make sense if we plan to change the build number scheme again. So it's all good and we may keep current implementation with -eq.

[the remark this answers is not in the document]
Comment (dpaulson45) [reply HealthChecker.ps1]: Even if we were planning on changing the build schema again, the current way would catch that providing we change the feature build number as well.

**Pull request merged**